frame/nomination-pools/src/lib.rs

```diff
@@ -1136,6 +1144,10 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type SubPoolsStorage<T: Config> = CountedStorageMap<_, Twox64Concat, PoolId, SubPools<T>>;
 
+	/// The minimum points to balance ratio that must be maintained for the pool to be `open`.
```

Comment:
needs an explanation of why `10` would mean a pool with `90%` slash

Reply:
How does the latest commit comment sound?

```
/// The minimum points to balance ratio that must be maintained for the pool to be `open`.
/// This is important in the event slashing takes place and the pool points to balance ratio
/// becomes disproportional.
/// For a value of 10, the threshold would be a pool points to pool balance ratio of 10:1.
/// Such a scenario would also be the equivalent of the pool being 90% slashed.
```